Description: Fix suspend quirks for Dell Inspiron 8600 with an ATI card
Author: Kees Cook <kees@ubuntu.com>
Index: pm-utils/video-quirks/20-video-quirk-pm-dell.quirkdb
===================================================================
--- pm-utils.orig/video-quirks/20-video-quirk-pm-dell.quirkdb	2010-03-17 22:45:21.964238428 +0100
+++ pm-utils/video-quirks/20-video-quirk-pm-dell.quirkdb	2010-03-17 23:02:37.791323516 +0100
@@ -53,6 +53,9 @@
    match system.hardware.product regex 1501|8600
     addquirk --quirk-s3-bios
     addquirk --quirk-s3-mode
+    match system.hardware.primary_video.vendor numeric_compare_eq 0x1002
+     addquirk --quirk-radeon-off
+    endmatch
    endmatch
    match system.hardware.product regex 9300
     addquirk --quirk-s3-bios
